neps/nep-0605-sharded-contracts.md

```diff
@@ -83,16 +93,16 @@ trait HostFunctions {
     // function panics.  If it was called using a new
     // `ShardedFunctionCallAction` then returns information about which global
     // contract code that is being used by the current account.
-    fn current_sharded_contract_info() -> ShardedContractInfo;
+    fn current_sharded_contract_info() -> Option<ShardedContractInfo>;
```

Comment:
Does this need to return an Option?  I propose that the host function panic in the case where it may have to return None.  

The smart contract knows whether or not it is running as a sharded contract.

Reply:
Yeah, that was my thinking as well.  Do you mind updating the PR to remove the `Option` and then I will merge it in.  We can then also add a discussion in the bottom about whether or not we should return an `Option`.

Reply:
removed this one

should I also remove it for `predecessor_sharded_contract_info`?